src/Microsoft.OpenApi/Models/OpenApiOperation.cs

```diff
@@ -255,7 +255,7 @@ public void SerializeAsV2(IOpenApiWriter writer)
                     {
                         var paramName = property.Key;
                         var paramSchema = property.Value;
-                        if (paramSchema.Type == "string" && paramSchema.Format == "binary") {
+                        if (paramSchema.Type == "string" && paramSchema.Format == "binary" || paramSchema.Format == "base64") {
```

Comment:
```c#
if (paramSchema.Type == "string" && paramSchema.Format == "binary" || paramSchema.Format == "base64")
```

perhaps you meant 

```c#
if (paramSchema.Type == "string" && (paramSchema.Format == "binary" || paramSchema.Format == "base64"))
```

or 

```c#
if ((paramSchema.Type == "string" && paramSchema.Format == "binary") || paramSchema.Format == "base64")
```

Reply:
var x = (false && (false || true)); // evaluates to false 
var x = (false && false || true); // evaluates to true

Adding the brackets makes it simpler to decipher the intention

Reply:
Yes I stand corrected.

Reply:
I also made a suggestion regarding the string comparisons themselves. The suggestion I made doesn't apply to the boolean algebra, and the expectation is that you'll combine both suggestions.